pkg/admin/graphql/identity_mutation.go

```diff
@@ -0,0 +1,65 @@
+package graphql
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/graphql-go/graphql"
+	"github.com/graphql-go/relay"
+
+	"github.com/authgear/authgear-server/pkg/lib/authn/identity"
+)
+
+var deleteIdentityInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "DeleteIdentityInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"identityID": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.ID),
+			Description: "Target identity ID.",
+		},
+	},
+})
+
+var deleteIdentityPayload = graphql.NewObject(graphql.ObjectConfig{
+	Name: "DeleteIdentityPayload",
+	Fields: graphql.Fields{
```

Comment:
GraphQL requires at least one fields in [object type](https://spec.graphql.org/June2018/#sec-Objects):
> An Object type must define one or more fields.

Echoing the deleted node may cause incorrect subsequent queries, since it may trigger database queries on non-existent entities.

Reply:
Can we name this field `_` and set the value as always false?

Reply:
Personally I'd prefer not. A `_` field is not meaningful, and removing it would be a breaking change, in case we want to add other fields in future such that the dummy field is not needed. A `success` field can always be kept and more meaningful than `_` field.

An alternative is to define a custom scalar `Void`, would you prefer that?

Reply:
@louischan-oursky Another idea is to return the parent user object, so that the client can query the updated identity list. I think that's a better approach for client-side caching of query results. What do you think?

Reply:
Let's keep `success` for now.

Reply:
Sure, let me revert the changes of returning parent user object.